### PR TITLE
created code_of_conduct file

### PR DESCRIPTION
**Pull request by Sloth-Panda**
this pr is related to issue #5

### CODE REVIEW

**Review by Salonii02 — APPROVED** [review]

**Review by manudd1011 — APPROVED** [review]

**Review by Salonii02 — APPROVED** [review]

**Review by manudd1011 — DISMISSED** [review]

**Comment by Sloth-Panda** [comment]
@manudd1011  can you please check out what's the issue is

**Review by praveenscience — APPROVED** [review]

File: Code_of_Conduct.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
+# Contributor Covenant Code of Conduct
```
Comment (praveenscience) [inline Code_of_Conduct.md:1]: Get rid of Covenant - that's a different community / company.

File: Code_of_Conduct.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+
+## Our Pledge
+
+In the interest of fostering an open and welcoming environment, we as
+contributors and maintainers pledge to making participation in our project and
+our community a harassment-free experience for everyone, regardless of age, body
+size, disability, ethnicity, sex characteristics, gender identity and expression,
+level of experience, education, socio-economic status, nationality, personal
+appearance, race, religion, or sexual identity and orientation.
```
Comment (praveenscience) [inline Code_of_Conduct.md:10]: Make it a single line.

File: Code_of_Conduct.md [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+
+## Our Standards
+
+Examples of behavior that contributes to creating a positive environment
+include:
```
Comment (praveenscience) [inline Code_of_Conduct.md:15]: Make it single line.

File: Code_of_Conduct.md [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+Examples of unacceptable behavior by participants include:
+
+* The use of sexualized language or imagery and unwelcome sexual attention or
+ advances
+* Trolling, insulting/derogatory comments, and personal or political attacks
+* Public or private harassment
+* Publishing others' private information, such as a physical or electronic
+ address, without explicit permission
+* Other conduct which could reasonably be considered inappropriate in a
+ professional setting
```
Comment (praveenscience) [inline Code_of_Conduct.md:32]: First and last two points in single line.

File: Code_of_Conduct.md [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+
+## Attribution
+
+This Code of Conduct is adapted from the [Contributor Covenant][homepage], version 1.4,
+available at https://www.contributor-covenant.org/version/1/4/code-of-conduct.html
+
+[homepage]: https://www.contributor-covenant.org
+
+For answers to common questions about this code of conduct, see
+https://www.contributor-covenant.org/faq
```
Comment (praveenscience) [inline Code_of_Conduct.md:76]: Let's rephrase a couple of things and let go off the attribution. I just spoke to a couple of open source communities and legal team, they said it can be rephrased to an extent it doesn't mean we copy.

Can you do it or do you need help with that?

**Review by praveenscience — CHANGES_REQUESTED** [review]

File: Code_of_Conduct.md (hunk 5 above)
Comment (Sloth-Panda) [reply Code_of_Conduct.md:76]: yes , i can do it but what should i particularly use for rephrasing

**Comment by manudd1011** [comment]
> @manudd1011 can you please check out what's the issue is

Issue resolved.

File: Code_of_Conduct.md [hunk 6]
```diff
@@ -1,56 +1,38 @@
...
+
 
 * Using welcoming and inclusive language
 * Being respectful of differing viewpoints and experiences
 * Gracefully accepting constructive criticism
 * Focusing on what is best for the community
 * Showing empathy towards other community members
 
-Examples of unacceptable behavior by participants include:
+Examples of unacceptable behavior by participants includes:
```
Comment (praveenscience) [inline Code_of_Conduct.md:18]: include, not includes.

File: Code_of_Conduct.md [hunk 7]
```diff
@@ -64,13 +46,3 @@ Further details of specific enforcement policies may be posted separately.
 Project maintainers who do not follow or enforce the Code of Conduct in good
 faith may face temporary or permanent repercussions as determined by other
 members of the project's leadership.
```
Comment (praveenscience) [inline Code_of_Conduct.md]: Single line please...

**Review by praveenscience — CHANGES_REQUESTED** [review]
Still we haven't changed a single character. We just removed the attribution. Now this is kind of stealing. Can we have this rephrased?

For example, if there's something:

> *"Mohit wants to buy milk"*

Can we rephrase it to:

> *"Raj wanted to buy chocolates"*

The context doesn't change. But the words change.

File: Code_of_Conduct.md (hunk 7 above)
Comment (Sloth-Panda) [reply Code_of_Conduct.md]: ok sir

**Comment by Sloth-Panda** [comment]
pr -> #12
